causalml/inference/tree/models.py

```diff
@@ -990,6 +1018,14 @@ def growDecisionTreeFrom(self, X, treatment, y, evaluationFunction, max_depth=10
                     rightScore2 = evaluationFunction(rightNodeSummary)
                     gain = (currentScore - p * leftScore1 - (1 - p) * rightScore2)
                     gain_for_imp = (len(X) * currentScore - len(X_l) * leftScore1 - len(X_r) * rightScore2)
+                elif evaluationFunction == self.evaluate_DDP:
+                    if self.control_name in leftNodeSummary and self.control_name in rightNodeSummary:
+                        leftScore1 = evaluationFunction(leftNodeSummary, control_name=self.control_name)
+                        rightScore2 = evaluationFunction(rightNodeSummary, control_name=self.control_name)
+                        gain = np.abs(leftScore1 - rightScore2)
+                        gain_for_imp = (len(X_l) * leftScore1 - len(X_r) * rightScore2)
```

Comment:
@jroessler I'm not familiar with this method, but just want to confirm if it's the case that `gain` takes the absolute difference while `gain_for_imp` doesn't

Reply:
@paullo0106 Thanks for the pointer! Actually it's a mistake! It should be the absolute value for `gain_for_imp` as well. I'll fix it and resubmit.